.github/workflows/runnable_cxx.yml

```diff
@@ -227,11 +227,15 @@ jobs:
           chmod +x ${TMP_CC}-wrapper ${TMP_CC}++-wrapper
         fi
 
+    - name: 'macOS 13: Switch to Xcode v14.3.1' # to work around '-macosx_version_min has been renamed to -macos_version_min' with some clang versions
```

Comment:
That error was encountered with clang v8, v9 and v13. These binaries from the LLVM GitHub repo weren't official/automated until very recently, i.e., some (obviously not totally random) guys uploaded their builds.